flake.nix

```diff
@@ -1,11 +1,84 @@
 {
-  description = "A very basic flake";
+  description = ""; # TODO
 
-  outputs = { self, nixpkgs }: {
+  inputs = {
+    utils.url = "github:numtide/flake-utils";
+    nixpkgs.url = "github:NixOS/nixpkgs";
+  };
 
-    packages.x86_64-linux.hello = nixpkgs.legacyPackages.x86_64-linux.hello;
+  outputs = { self, nixpkgs, utils }:
+    utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = nixpkgs.legacyPackages."${system}";
+        src = pkgs.fetchFromGitHub {
+          owner = "decentralized-identity";
+          repo = "universal-resolver";
+          rev = "5c2147f7992dd2c622aaf659309131e4cb881c05";
+          sha256 = "0y1jg5aybnkw450k14wpy92mgmnnvr1ayym98a70nx3fbjgad4md";
+        };
```

Comment:
This should be an input to the flake itself, that way, updating the source is as simple as `nix flake update`.

Reply:
Done. Thanks!